### PR TITLE
Create mouseFPV_ImpuseRC_Apex_tune.txt

### PR DESCRIPTION
**Pull request by rochford77**
Custom Tune for ImpulseRC Apex Frame. Experimental.

### CODE REVIEW

File: presets/4.4/tune/mouse_fpv/mouseFPV_ImpuseRC_Apex_tune.txt [hunk 1]
```diff
@@ -0,0 +1,205 @@
+#$ TITLE:  ImpulseRC Apex | mouseFPV
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: Apex, ImpulseRC
```
Comment (limonspb) [inline presets/4.4/tune/mouse_fpv/mouseFPV_ImpuseRC_Apex_tune.txt:5]: should add keywords for prop size like 5" 5in, 5 inch?

File: presets/4.4/tune/mouse_fpv/mouseFPV_ImpuseRC_Apex_tune.txt [hunk 2]
```diff
@@ -0,0 +1,205 @@
+#$ TITLE:  ImpulseRC Apex | mouseFPV
```
Comment (limonspb) [inline presets/4.4/tune/mouse_fpv/mouseFPV_ImpuseRC_Apex_tune.txt:1]: should add like ImpulseRC Apex 5"  | mouseFPV?

**Comment by rochford77** [comment]
@limonspb thanks for the feedback. Yeah, I suppose it would be good to know prop size haha, especially since there are multiple versions of the Apex. It has been a min since I have amended a commit, hopefully I did that right, you should be able to see the changes you requested in ec202b1 :-)

**Comment by limonspb** [comment]
> @limonspb thanks for the feedback. Yeah, I suppose it would be good to know prop size haha, especially since there are multiple versions of the Apex. It has been a min since I have amended a commit, hopefully I did that right, you should be able to see the changes you requested in [ec202b1](https://github.com/betaflight/firmware-presets/commit/ec202b17de07655a8b35249ba76af6566c6dacae) :-)

looks good.
and now you don't have to squash or amend anymore.
Because now maintainer can do that when he merges the PR :)

**Review by limonspb — APPROVED** [review]

**Comment by der-ray** [comment]
I am using the less cheese version on an o3 APEX with an AUW of ~650g and it is pretty damn good imho. Just came by to say thank you for the preset :)

**Comment by rochford77** [comment]
> I am using the less cheese version on an o3 APEX with an AUW of ~650g and it is pretty damn good imho. Just came by to say thank you for the preset :)

Hey thanks, that's good to hear! I appreciate the feedback it definitely helps to know what's working and what isn't.

**Comment by alexberdnik** [comment]
My 2c:
The preset works fine on apex evo o3 (650g), with 8k pidloop, jhemcu f722 controller.
Unfortunately it doesn't work that well for the same frame/motors/weight but with 4k pidloop (speedybee f405v4 controller).
Haven't tried the "less spicy" version yet.

**Comment by alexberdnik** [comment]
> My 2c: The preset works fine on apex evo o3 (650g), with 8k pidloop, jhemcu f722 controller. Unfortunately it doesn't work that well for the same frame/motors/weight but with 4k pidloop (speedybee f405v4 controller). Haven't tried the "less spicy" version yet.

turned out it was a faulty FC. Works great with antoher f405, not spedybee though

**Pull request merged**